gsdk/src/signer/utils.rs

```diff
@@ -33,4 +39,64 @@ impl Signer {
 
         Ok(())
     }
+
+    /// Run transaction.
+    ///
+    /// This function allows us to execute any transactions in gear.
+    ///
+    /// # You may not need this.
+    ///
+    /// Read the docs of [`Signer`] to checkout the wrappred transactions,
+    /// we need this function only when we want to execute a transaction
+    /// which has not been wrapped in `gsdk`.
+    ///
+    /// # Example
+    ///
+    /// ```ignore
+    /// use gsdk::{
+    ///   Api,
+    ///   Signer,
+    ///   metadata::calls::BalancesCall,
+    ///   Value,
+    /// };
+    ///
+    /// let api = Api::new(None).await?;
+    /// let signer = Signer::new(api, "//Alice", None).await?;
+    ///
+    /// {
+    ///     let args = vec![
+    ///         Value::unnamed_variant("Id", [Value::from_bytes(dest.into())]),
+    ///         Value::u128(value),
+    ///     ];
+    ///     let in_block = signer.run_tx(BalancesCall::Transfer, args).await?;
+    /// }
+    ///
+    /// // The code above euqals to:
+    ///
+    /// {
+    ///    let in_block = signer.transfer(dest, value).await?;
```

Comment:
I'm not pretty sure that it affects api positively. I'd go with
`signer.calls.transfer`
and
`signer.storage.number`
to split functionality and make it more readable + intuitive

Reply:
#2895